src/sdk/pynni/nni/compression/torch/builtin_pruners.py

```diff
@@ -26,7 +27,7 @@ def __init__(self, model, config_list):
         """
 
         super().__init__(model, config_list)
-        self.if_init_list = {}
+        self.mask_calculated_ops = set()
```

Comment:
better to add comments to explain this variable

Reply:
resolved